_sources/Strings/Exercises.rst

```diff
@@ -105,36 +105,28 @@ Exercises
 
            .. actex:: ex_8_3
 
-              Assign to a variable in your program a triple-quoted string that contains
-              your favorite paragraph of text - perhaps a poem, a speech, instructions
-              to bake a cake, some inspirational verses, etc.
+                Assign to a variable in your program a triple-quoted string that contains
+                your favorite paragraph of text - perhaps a poem, a speech, instructions
+                to bake a cake, some inspirational verses, etc.
    
-              Write a function that counts the number of alphabetic characters (a through z, or A through Z) in your text and then keeps track of how many are the letter 'e'.  Your function should print an analysis of the text like this::
+                Write a function that counts the number of alphabetic characters (a through z, or A through Z) in your text and then keeps track of how many are the letter 'e'.  Your function should print an analysis of the text like this::
    
-                  Your text contains 243 alphabetic characters, of which 109 (44.8%) are 'e'.
-              ~~~~
-              def count(p):
-                  # your code here
-                
-         
-              from unittest.gui import TestCaseGui
-
-              class myTests(TestCaseGui):
-                  def testOne(self):
-                      string1 = "e"
-                      string2 = "eieio"
-                      string3 = "eeeeeeeeeeee"
-                      string4 = "elephant"
-                      self.assertEqual(count(string1), 1, "Just one e")
```

Comment:
somehow these extra tests did not get added to the final version.

Reply:
I just fetched upstream again to update this branch, did that fix it?

Reply:
looks like it didn't

Reply:
This has been fixed thanks to Brad's help to update my master branch